reference/commands/consumer/config.rst

```diff
@@ -129,19 +129,25 @@ user has in the local machine will be deleted.
 
 All the configuration files will be copied to the Conan home directory. These are the special files and the rules applied to merge them:
 
-+--------------------------------+----------------------------------------------------------------------+
-| File                           | How it is applied                                                    |
-+================================+======================================================================+
-| profiles/MyProfile             | Overrides the local ~/.conan/profiles/MyProfile if already exists    |
-+--------------------------------+----------------------------------------------------------------------+
-| settings.yml                   | Overrides the local ~/.conan/settings.yml                            |
-+--------------------------------+----------------------------------------------------------------------+
-| remotes.txt                    | Overrides remotes. Will remove remotes that are not present in file  |
-+--------------------------------+----------------------------------------------------------------------+
-| config/conan.conf              | Merges the variables, overriding only the declared variables         |
-+--------------------------------+----------------------------------------------------------------------+
-| hooks/my_hook.py               | Overrides the local ~/.conan/hooks/my_hook.py if already exists      |
-+--------------------------------+----------------------------------------------------------------------+
++--------------------------------+-----------------------------------------------------------------------------+
+| File                           | How it is applied                                                           |
++================================+=============================================================================+
+| profiles/MyProfile             | Overrides the local ~/.conan/profiles/MyProfile if already exists           |
++--------------------------------+-----------------------------------------------------------------------------+
+| settings.yml                   | Overrides the local ~/.conan/settings.yml                                   |
++--------------------------------+-----------------------------------------------------------------------------+
+| remotes.txt                    | Overrides remotes. Will remove remotes that are not present in file         |
++--------------------------------+-----------------------------------------------------------------------------+
+| remotes.json (**Since 1.52**)  | | Overrides remotes. Will remove remotes that are not present in file.      |
```

Comment:
Is this rendered correctly with the double ``| |``?

Reply:
Yes, it's the way to have multiline text in the table

![image](https://user-images.githubusercontent.com/5045666/187362646-365f2a60-274b-4938-a1ef-d59d1b7220e8.png)